CHANGELOG.md

```diff
@@ -1,3 +1,42 @@
+## 10.0.0
+
+Another major release! We have some great improvements in this version but also have breaking changes. Please review the following to see if you need to change anything to upgrade `zapier-platform-core` to v10.
+
+First, Zapier integrations that depend on the new Core v10 **will run using Node.js 12**. You can install Node.js 12 using `nvm`.
```

Comment:
I would list each of these bolded breaking changes in the bullets below. I like these wordier summaries, but it should also be easy to scan a list of bullets for ❗and quickly know what, if anything, they need to do to run their app on v10.